examples/GPU_Examples/NonCartesian_gpuNUFFT_DensityCompensation.py

```diff
@@ -0,0 +1,74 @@
+"""
+Neuroimaging non-cartesian reconstruction
+=========================================
+
+Author: Chaithya G R
+
+In this tutorial we will reconstruct an MRI directly with density compensation
```

Comment:
MRI image or MR image

Reply:
don't forget to port all the changes from this example to the other one (SENSE)

Reply:
Yes on it